### PR TITLE
Add Uninstall section

### PR DESCRIPTION
**Pull request by eoq**
Just a quick instruction on how to remove an AWX deployment from a new user who struggled with it for a bit.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -760,6 +761,17 @@ Example configuration of environment variables
       eks.amazonaws.com/role-arn: arn:aws:iam::<ACCOUNT_ID>:role/<IAM_ROLE_NAME>
 ```
 
+
+### Uninstall ###
+
+To uninstall the AWX deployment you need to direct AWX Operator to initiate the removal which will delete all AWX objects except for persistent volumes.
```
Comment (tchellomello) [inline README.md]: Well, that is not completely true if the user has the `garbage_collect_secrets: true` on the SPEC. So maybe we could rephrase this statement for:

_To uninstall an AWX deployment instance, you basically need to remove the `AWX` kind related to that instance. For example, to delete a `awx` instance named `awx-demo`, you would do:_

```bash
$ kubectl delete awx awx-demo
awx.awx.ansible.com "awx-demo" deleted
```

_Deleting an `AWX` instance will remove all deployments and statefulsets related, however, `persistence volumes `and `secrets` will remain. To enforce `secrets` get also removed, you can use the `garbage_collect_secrets: true`._

**Review by tchellomello — CHANGES_REQUESTED** [review]
Thanks for your contribution @eoq 

I added some notes on your PR. I don't want to be too prescriptive. You can rephrase the text if you want, however,  let's make sure a similar message is there. 

Thank you

File: README.md (hunk 1 above)
Comment (eoq) [reply README.md]: Great. I will update it per your advise.

File: README.md [hunk 2]
```diff
@@ -760,6 +761,18 @@ Example configuration of environment variables
...
+
+To uninstall an AWX deployment instance, you basically need to remove the AWX kind related to that instance. For example, to delete an AWX instance named awx-demo, you would do:
+
+```bash
+$ kubectl delete awx awx-demo
+awx.awx.ansible.com "awx-demo" deleted
+```
+
+Deleting an AWX instance will remove all related deployments and statefulsets, however, persistent volumes and secrets will remain. To enforce secrets also getting removed, you can use garbage_collect_secrets: true.
+
```
Comment (tchellomello) [inline README.md:775]: Could you please add the `garbage_collect_secrets: true` between single-quote?

**Review by tchellomello — CHANGES_REQUESTED** [review]

**Review by tchellomello — APPROVED** [review]
Thanks for working on this @eoq 

We really appreciate your assistance with the project!

**cc**: @rooftopcellist let's see what @rooftopcellist  says about and then we can merge it!

**Review by rooftopcellist — APPROVED** [review]
Thank you for contributing this to the docs @eoq !   These notes look good to me.

**Pull request merged**